src/Trainer10.py

```diff
@@ -540,7 +540,7 @@ def get_optimizers(configs, model):
 
 def start_training(configs: dict):
     root_dirpath = Path('../')
-    database_dirpath = root_dirpath / 'data/databases' / configs['database_dirpath']
+    database_dirpath = root_dirpath / 'data' / configs['database_dirpath']
```

Comment:
configs['database_dirpath'] should be `NeRF_LLFF/data` and not `databases/NeRF_LLFF/data`. This is because the `databases` prefix is common across datasets, no need to to include it for every database. So, this line should have `root_dirpath / 'data/databases' / configs['database_dirpath']`